mkdocs/utils.py

```diff
@@ -216,12 +214,10 @@ def create_relative_media_url(nav, url):
     return relative_url
 
 
-def normalise_path(path):
-    """
-    Normalise POSIX and NT paths to be consistently POSIX style.
-    """
+def path_to_url(path):
+    """Convert a system path to a URL."""
 
-    if ntpath.sep in path:
-        path = path.replace(ntpath.sep, posixpath.sep)
+    if os.path.sep == '/':
+        return path
 
-    return path
+    return pathname2url(path)
```

Comment:
I think perhaps this would be best:

```python
def path_to_url(path):
    """Convert a system path to a URL."""
    if os.sep != '/':
        path = path.replace(os.sep, '/')
    return pathname2url(path)
```